tests/workloads/megatron_run/test_megatron_run.py

```
@@ -24,6 +24,51 @@
 from cloudai.workloads.megatron_run import MegatronRunCmdArgs, MegatronRunTestDefinition
 
 
+class TestMegatronRunCmdArgs:
+    BASE_ARGS = dict(docker_image_url="http://url", run_script=Path(__file__))
```

Ruff (0.15.21)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)